WordPress/WordPressTest/ReaderDetailViewControllerTests.swift

```diff
@@ -27,7 +27,7 @@ class ReaderDetailViewControllerTests: XCTestCase {
         super.tearDown()
     }
 
-    func testReblogButonMathchesExpectation() {
+    func testReblogButtonMathchesExpectation() {
```

Comment:
One more time - `Mathches` should be `Matches`.

Reply:
Sorry about that! Looks like I'm blind today